nextflow.config

```diff
@@ -1,81 +1,21 @@
 params {
     run_id = 1
     ref_ver = "hg19"
-    ref_dir = " "
-    bed_filter = " "
+    ref_dir = null
+    bed_filter = null
```

Comment:
can you confirm if it works well with "null"?
I am not sure, but I remember nextflow was not working well with "null"
there must be a reason in the existing implementation.